examples/advanced-ops-outbrain/01-Download-Convert.ipynb

```diff
@@ -116,23 +119,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from cudf import read_csv\n",
-    "\n",
     "# use managed memory for device memory allocation\n",
     "rmm.reinitialize(managed_memory=True)\n",
     "\n",
     "# Merge all the CSV files together\n",
-    "documents_meta = read_csv(DATA_BUCKET_FOLDER + \"documents_meta.csv\", na_values=[\"\\\\N\", \"\"])\n",
+    "documents_meta = df_lib.read_csv(DATA_BUCKET_FOLDER + \"documents_meta.csv\", na_values=[\"\\\\N\", \"\"])\n",
```

Comment:
instead of calling df_lib everywhere why not import read_csv from df_lib after its set?

Reply:
I agree it would be nice. I tried doing that, but I kept getting this error
```
>>> from nvtabular.dispatch import get_lib
>>> df_lib = get_lib()
>>> from df_lib import read_csv
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'df_lib'
```

It seems it tried to use the variable name as a library itself, instead of whatever library it is pointing. I spent some time looking online, but I was not able to find how to do it successfully. Do you have some idea?

Reply:
Can you create a second alias and move that around? i.e 
```
df_lib = get_lib()
read_csv = df_lib.read_csv
```

Reply:
This actually works. Thanks Julio